building/buildah_speed/buildah_speed.sh

```diff
@@ -0,0 +1,165 @@
+#!/bin/sh
+
+# buildah_speed.sh demo script.
+# This script will demonstrate at an speed
+# for Buildah command running inside of a container
+# the Podman packages must be installed.
+
+ 
+# Setting up some colors for helping read the demo output.
+# Comment out any of the below to turn off that color.
+bold=$(tput bold)
+cyan=$(tput setaf 6)
+reset=$(tput sgr0)
+
+read_color() {
+    read -p "${bold}$1${reset}"
+}
+
+echo_color() {
+    echo "${cyan}$1${reset}"
+}
+
+setup() {
+    rpm -q podman >/dev/null
+    if [[ $? != 0 ]]; then
+	echo $0 requires the podman package to be installed
+	exit 1
+    fi
+    rpm -q buildah >/dev/null
+    if [[ $? != 0 ]]; then
+	echo $0 requires the buildah package to be installed
+	exit 1
+    fi
+    sudo dnf -y update --downloadonly 2>/dev/null
```

Comment:
need to add an echo here letting people know what's going on and that it will take a bit.

Reply:
Fixed

Reply:
I don't see the fix....

Reply:
echo_color "Setting up Demo"